graphql/build.gradle

```
@@ -26,6 +17,12 @@ dependencies {
 	api project(':' + rootProject.name + '-auth')
 }
 
+// Javadoc
+javadoc {
+	// Ignore auto-generated files from apollo graphql
+	exclude 'com/github/twitch4j/graphql/internal/**'
+}
+
```

Because about javadoc aggregation you need change this like that.
```suggestion
tasks.withType<Javadoc> {
	// Ignore auto-generated files from apollo graphql
	exclude 'com/github/twitch4j/graphql/internal/**'
}
```